x-pack/legacy/plugins/spaces/public/views/management/components/confirm_delete_modal.tsx

```diff
@@ -122,17 +122,17 @@ class ConfirmDeleteModalUI extends Component<Props, State> {
               <EuiFormRow
                 label={intl.formatMessage({
                   id: 'xpack.spaces.management.confirmDeleteModal.confirmSpaceNameFormRowLabel',
-                  defaultMessage: 'Confirm space name to delete',
+                  defaultMessage: 'Please type DELETE to delete your space.',
```

Comment:
```suggestion
                  defaultMessage: 'Please type DELETE to confirm this action.',
```
Minor nit: could you change the message so it still reflects that this is a confirmation prompt?

Reply:
According to our EUI writing guidelines, we typically avoid the word "please". Maybe also include the why before the how. For example:

To confirm, enter DELETE.